client/me/help/help-contact-form/index.jsx

```diff
@@ -512,12 +512,16 @@ export class HelpContactForm extends PureComponent {
 				  } );
 
 		const hasNoSites = siteCount === 0;
+		const helpSiteIsNotWpCom = analyseSiteData && analyseSiteData.startsWith( 'isNonWpComHosted' );
 
 		let noticeMessage;
 		let actionLink;
 		let actionMessage;
 
-		if ( analyseSiteData === 'isWpComConnectedSiteNotLinkedToAccount' ) {
+		if (
+			showAlternativeSiteOptionsField &&
```

Comment:
Since I'm re-using the `<Notice>` component for the new message I had to move it outside the `showAlternativeSiteOptionsField` condition [here](https://github.com/Automattic/wp-calypso/pull/62656/files#diff-a13cc8750e3ac74612ac3ada223def7f04e4fae95832f180eca78e294ec11195L611)

I added the condition here to ensure the message is displayed only to users in the same conditions as before.

Reply:
Ah, sorry, I missed that 🤦 Carry on...
Though, it still seems like we'd want to review these (after this PR is merged), as it seems this notice could be useful on non-default support levels as well.